src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,91 @@
+package parser;
+
+import commands.*;
```

Comment:
Just a minor note!  
  
Imported classes should always be listed explicitly.  
[coding standard](https://se-education.org/guides/conventions/java/intermediate.html#package-and-import-statements)

Reply:
Thank you for pointing that out. Apparently my IDE does auto-wildcard import when 5 imports are made. I have since disabled that feature :smile: